src/fonduer/learning/disc_learning.py

```diff
@@ -248,19 +249,18 @@ def train(
                     or epoch in [0, (self.settings["n_epochs"] - 1)]
                 )
             ):
-                msg = "[{0}] Epoch {1} ({2:.2f}s)\tAverage loss={3:.6f}".format(
-                    self.name,
-                    epoch + 1,
-                    time() - st,
-                    torch.stack(iteration_losses).mean(),
+                msg = (
+                    f"[{self.name}] "
+                    f"Epoch {epoch + 1} ({time() - st:.2f}s)\t"
+                    f"Average loss={torch.stack(iteration_losses).mean():.6f}"
                 )
                 if X_dev is not None:
                     scores = self.score(
                         _X_dev, Y_dev, batch_size=self.settings["batch_size"]
                     )
                     score = scores if self.cardinality > 2 else scores[-1]
                     score_label = "Acc." if self.cardinality > 2 else "F1"
-                    msg += "\tDev {0}={1:.2f}".format(score_label, 100.0 * score)
+                    msg += f"\tDev {score_label}={100.0 * score:.2f}"
```

Comment:
Should double check that the order of operations here results in the correct formatting.

i.e., does this correctly execute as 
```py
(100.0 * score):.2f
```
?

Reply:
Checked, this works.